common/djangoapps/request_cache/tests.py

```diff
@@ -0,0 +1,20 @@
+"""
+Tests for the request cache.
+"""
+from django.conf import settings
+from django.test import TestCase
+
+from request_cache import get_request_or_stub
+
+
+class TestRequestCache(TestCase):
+    """
+    Tests for the request cache.
+    """
+
+    def test_get_request_or_stub(self):
+        # Outside the context of the request, we should still get a request
+        # that allows us to build an absolute URI.
+        stub = get_request_or_stub()
+        expected_url = "http://{site_name}/foobar".format(site_name=settings.SITE_NAME)
```

Comment:
We can't determine from Django settings whether there's a proxy doing HTTPS --> HTTP translation for us.  HTTP is a safer default, since it's configured to redirect to HTTPS in prod.